resources/shaders/rt_hybrid.hlsl

```diff
@@ -161,28 +168,21 @@ void RaygenEntry()
 
 	if (length(normal) == 0)		//TODO: Could be optimized by only marking pixels that need lighting, but that would require execute rays indirect
 	{
-		gOutput[DispatchRaysIndex().xy] = float4(skybox.SampleLevel(s0, SampleSphericalMap(-V), 0));
+		// A value of 1 in the output buffer, means that there is shadow
+		// So, the far plane pixels are set to 0
+		output_refl_shadow[DispatchRaysIndex().xy] = float4(0, 0, 0, 0);
 		return;
 	}
 
-	float3 lighting = shade_pixel(wpos, V, albedo, metallic, roughness, normal, rand_seed, 0);
-	float3 reflection = DoReflection(wpos, V, normal, rand_seed);
-
-	float3 flipped_N = normal;
-	flipped_N.y *= -1;
-	const float3 sampled_irradiance = irradiance_map.SampleLevel(s0, flipped_N, 0).xyz;
+	wpos += normal * EPSILON;
```

Comment:
Here. Is duplicate applying the epsilon * normal twice.

Reply:
Fixed